src/os/bluestore/NVMEDevice.cc

```diff
@@ -201,10 +198,13 @@ class SharedDriverData {
         flush_lock("NVMEDevice::flush_lock"),
         flush_waiters(0),
         completed_op_seq(0), queue_op_seq(0) {
+    enum spdk_nvme_qprio qprio = SPDK_NVME_QPRIO_URGENT;
```

Comment:
Yes,  the default is URGENT(0). There are three values defined in SPDK, and the default priority is 0. It is correctly used in this scenario